package.json

```diff
@@ -59,14 +59,15 @@
   "types": "./index.d.ts",
   "scripts": {
     "lint": "populist lib/*.js lib/winston/*.js lib/winston/**/*.js",
-    "pretest": "npm run lint",
```

Comment:
Make this `npm run lint` instead of removing it.

Reply:
I'm not sure why we were linting files prior to running the tests, as by no means is it a required step. This is why I removed it. Subsequently, linting is a part of the CI checks already. 

What is it exactly you want me to do instead?

Reply:
Oh nevermind. I misread here.